extensions/wikia/Chat2/Chat.class.php

```diff
@@ -346,11 +340,12 @@ public static function clearBanInfoCache( $wikiID, $userID ) {
 	}
 
 	private static function getBanInfoCacheKey( $wikiID, $userID ) {
-		return wfMemcKey( 'chat-baninfo', $wikiID, $userID );
+		// Using shared mem key, but adding in the WikiID ourselves since its possible
+		// to call these functions with an alternate wiki ID.
+		return wfSharedMemcKey( 'chat-baninfo', $wikiID, $userID );
```

Comment:
I still don't understand, why are we using `wfSharedMemcKey` here? If we're passing in the `$wikiID` every time, isn't that essentially the same thing as `wfMemcKey`?